types/field_name.go

```diff
@@ -28,7 +28,8 @@ type FieldName struct {
 	TblName     model.CIStr
 	ColName     model.CIStr
 
-	Hidden bool
+	Hidden        bool
+	NotAccessible bool
```

Comment:
`NotAccess` is not a good name here, for update statement, it can also write the `write only` column implicitly (like adding default value)(can access to it), the meaning here is to emphasize that condition checker should turn blind to `write only` columns.

write can see it, read can't see it.

Reply:
PTAL again, I changed the name and add a comment